_config.yml

```diff
@@ -201,13 +201,13 @@ authors:
     about: "Here's a little about shwetashambhavi"
 
   spaceranger-23:
-    name: spaceranger-23
+    name: Reginald Evans 
     prof: false
     gravatar:
     website: "#"
     github: spaceranger-23
-    twitter:
-    about: "Here's a little about spaceranger-23"
+    twitter: RE3GIE
+    about: "Reginlad is a second year student with the MSIS program at the University of Texas at Austin. His career aspirations consist of becoming a cyber security/software developer in the near future. In his off time, Reginald likes spend his time running at outdoors trails and with his family."
```

Comment:
This all looks like it is formatted perfectly! Looks like there's a typo in your name here. It won't hurt anything, but I wanted to give you a chance to change it if that's important to you.

Reply:
@MishaCienfuegos thanks for pinpointing that out. I had to spell my name Re3GIE on twitter due to there being thousands of Reggie's in the world. lol I am good to go for submission if you are. Thanks